### DISCUSSION TITLE
Show tree in Raw Message panel for small fixed numeric arrays

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#162] [Q&A]
In the attached bag, using the Raw Message panel I can set the message path to `/gps/fix.position_covariance[:]` and see the list of all nine matrix cells. But if I use the topic only (`/gps/fix`) that field renders as

> Float64Array(9) [0.011881, 0, 0, 0, 0.011881, 0, 0, 0, 0.009409] 

With no ability to plot individual values. Since fixed arrays of length 3/4/9/16 are common in ROS for holding vectors and matrices, it would be nice if these showed up in a "more native" way in Studio vs the currently handling which seems more suited for large data buffers like image or point cloud data.

<img width="450" alt="Screen Shot 2021-10-21 at 4 18 02 PM" src="https://user-images.githubusercontent.com/195374/138369596-ca633ce8-8f69-4a26-a994-320535961e2e.png">

<img width="452" alt="Screen Shot 2021-10-21 at 4 18 11 PM" src="https://user-images.githubusercontent.com/195374/138369606-eb011179-ab1d-47e5-b855-e6ffa7a54e1d.png">

[gps.bag.zip](https://github.com/foxglove/studio/files/7393362/gps.bag.zip)

**Comment by defunctzombie** [comment]
> With no ability to copy-paste

Could you clarify this? I was able to copy the values directly from the RawMessage panel by selecting all the values between the `[` `]` in the topic version.

**Comment by defunctzombie** [comment]
You can also click the "Copy msg" icon to copy the entire message.

**Comment by defunctzombie** [comment]
Related change to make copying just an array easier:
https://github.com/foxglove/studio/pull/2016

**Comment by jhurliman** [comment]
@defunctzombie updated to only reference plotting.
